src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div>
+      <Display />
+      <ButtonPanel />
+    </div>
```

Comment:
- [ ] Please wrap this in a `React.Fragment` rather than a `div`. There is no need for the extra `div` node.